src/NuGet.Clients/PackageManagement.UI/Actions/UIActionEngine.cs

```diff
@@ -633,16 +633,16 @@ private static IReadOnlyList<PreviewResult> GetPreviewResults(IEnumerable<Resolv
         /// Get the package metadata to see if RequireLicenseAcceptance is true
         /// </summary>
         private async Task<List<IPackageSearchMetadata>> GetPackageMetadataAsync(
-            INuGetUI uiService,
+            IEnumerable<SourceRepository> sources,
             IEnumerable<PackageIdentity> packages,
             CancellationToken token)
         {
             var results = new List<IPackageSearchMetadata>();
 
             // local sources
-            var sources = new List<SourceRepository>();
-            sources.Add(_packageManager.PackagesFolderSourceRepository);
-            sources.AddRange(_packageManager.GlobalPackageFolderRepositories);
+            var localSources = new List<SourceRepository>();
```

Comment:
you updated the variable name but didn't change the name at the usage.  Update `sources` at line 652